oshi-core/src/main/java/oshi/software/os/mac/MacNetworkParams.java

```diff
@@ -30,6 +30,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.sun.jna.Native;
+import com.sun.jna.platform.unix.LibC;
```

Comment:
I was going to suggest the Mac `SystemB` class here, but turns out it doesn't import from `LibC` (like it should) so it seems I need to submit a change to JNA now.  :)  (so this can stay as is for now)

Reply:
Actually, this doesn't work if the user doesn't have `gcc` installed.  So change this to `SystemB` and add an `extends LibCAPI` (importing from ` com.sun.jna.platform.unix.LibCAPI`) to OSHI's `SystemB` interface.